comment/conf/defaults.py

```diff
@@ -28,3 +28,5 @@
 COMMENT_ALLOW_TRANSLATION = False
 
 COMMENT_ALLOW_SUBSCRIPTION = False
+
+COMMENT_ORDER_BY = '-posted'
```

Comment:
Why are you limiting this to one value?
```suggestion
COMMENT_ORDER_BY = ['-posted']
```
We can use list of values and unpack it in `order_by`
https://docs.djangoproject.com/en/3.1/ref/models/querysets/#order-by

Reply:
valildation could get a bit tricky, and it would be tough to explain the documentation, that is why i avoided this at first.

Suppose, we allow a tuple and check if each value of the tuple is present in allowed ones, something like this would also escape this: `('posted', '-posted')` and many more like these. 

Linking to the above `django` documentation(one that you pointed) is an option, but they allow a lot of other things such as model's relational fields, query expressions, etc.

Another option is just to ignore this. This sort of thing isn't quiet explained well even in the `django` documentation(at least I couldn't see it). 

I couldn't quite think of an elegant way to solve this. If you have suggestions it would be great to know them.